neqo-transport/src/shuffle.rs

```diff
@@ -121,6 +124,19 @@ mod tests {
         assert!(super::find_sni(truncated).is_none());
     }
 
+    #[test]
+    fn find_sni_invalid_sni_length() {
+        // ClientHello with an SNI extension with an invalid length
+        let mut buf = Vec::from(BUF_WITH_SNI);
+        let len = buf.len();
+
+        assert!(buf[len - 23] == 0x00 && buf[len - 22] == 0x0c); // Check Server Name List length
```

Comment:
```suggestion
        assert_eq!(&buf[len-23..len-22], &[0x00, 0x0c], "check SNI length");
```

...will fix your odd comment shape at the same time